docroot/modules/custom/webform_serialization/src/Normalizer/WebformNormalizer.php

```diff
@@ -36,4 +38,20 @@ public function getFields($entity, $bundle, ResourceType $resource_type) {
     return $enabled_public_fields;
   }
 
+  /**
+   * Add the options for select items if the options are stored in config files.
+   *
+   * @param array $formItems
+   *   The already parsed form array.
+   */
+  private function getSelectFields(array &$formItems) {
```

Comment:
Unless there's a strong reason to make this private, let's make it protected in case we ever extend this class and need to access this method from there.